src/parser.ts

```diff
@@ -22,10 +22,13 @@ type NotesTree = {
 
 // `?<=` to except `(^|\s)#` it self
 //
-// The `[\s]?` is to require a hashtag (or nested hashtags) must followed by a whilespace character
+// The `[\s]?` is to require a hashtag (or nested hashtags) must followed by a whitespace character
 // or at the end of the line, in order to avoid https://github.com/vanadium23/markdown-hashtags/issues/13.
 // Do not forget to use `.trimEnd()` in the code.
-export const hashtagRegexp = /(?<=(^|\s)#)[^\s!@#$%^&*()=+.,\[{\]};:'"?><]+[\s]?/g;
+
+// #24 in addition to "/" also "." can be used as tag separator
+// export const hashtagRegexp = /(?<=(^|\s)#)[^\s!@#$%^&*()=+.,\[{\]};:'"?><]+[\s]?/g;
```

Comment:
Don't bother yourself with commented code, we have a VCS for that. You can just change line. 👍

Reply:
I did this to check first, if it works as required. Will be removed, when all works fine.